Components/cards.css

```diff
@@ -0,0 +1,168 @@
+:root{
+    --card-violet : #973a99;
+    --card-gray : #808080;
+    --card-orange : #ffa500;
+    --card-white : #ffffff;
+    --card-box-shadow :  1px 4px 10px 1px #3b3f4266; 
+}
+
+
+/*--------Utilities--------*/
+.card-rating{
+    display: flex;
```

Comment:
Since you are using flex property many a times in your code, why not create a utility class and use wherever required.

Reply:
I thought so but that would make few of my html elements to have too many classes. So I dropped that. But yes thanks for your suggestion will try to implement it.